### PR TITLE
[Refactor] Add type hints to `geomap.py` and a small test refactor

### PR DESCRIPTION
**Pull request by dshemetov**
### Description
- Add type hints and improves formatting in `geomap.py`
- Refactors a few other tests in `_delphi_utils`

### Changelog
You can see the changes in the commit log.

Also adds `.git-blame-ignore-revs`, which you can use to ignore the changes in the `83cb164` commit (or whatever commits you paste into it) when using `git blame` (idea taken from [`black` documentation](https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html)). If you're a GitLens user, [here's the solution](https://github.com/gitkraken/vscode-gitlens/issues/413#issuecomment-523640184).

### CODE REVIEW

File: .git-blame-ignore-revs
Comment (krivard) [inline .git-blame-ignore-revs:1]: note to future travelers this is so that git-annotate skips Black (purely formatting) commits

**Review by krivard — COMMENTED** [review]
should we refresh this or trash it?

**Comment by dshemetov** [comment]
I'd say refresh. I did this because I was working geomapper at some point and the lack of argument types was confusing my IDE.

**Comment by krivard** [comment]
alright, when you get a minute, rebase onto main to bring it back up to date, then i'll tag nat for a review

**Review by nmdefries — APPROVED** [review]
I don't see the `.git-blame-ignore-revs` file -- did you mean to remove that from the PR?

Aside from that, looks good!

**Comment by nmdefries** [comment]
praise: good job finishing this off!

**Pull request merged**